client/state/signup/steps/site-vertical/selectors.js

```diff
@@ -4,12 +4,49 @@
  * External dependencies
  */
 
-import { get } from 'lodash';
+import { get, find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getVerticals } from 'state/signup/verticals/selectors';
+import { DEFAULT_VERTICAL_KEY } from 'state/signup/verticals/constants';
 
 export function getSiteVerticalName( state ) {
 	return get( state, 'signup.steps.siteVertical.name', '' );
 }
 
+export function getSiteVerticalData( state ) {
+	const verticalName = get( state, 'signup.steps.siteVertical.name', '' );
```

Comment:
Why not use `getSiteVerticalName( state )`?

Reply:
Correct. Applied in 72356b1.